components/brave_rewards/resources/page/components/settings.tsx

```diff
@@ -54,6 +72,61 @@ export function Settings () {
     return false
   }
 
+  class HashHandler {
+    hash: string;
+    dataDeeplinkId: string;
+    action: any;
```

Comment:
Can we avoid `any` here? I guess this would have to be

```
() => Action<any> | null
```

based on the constructor type?

Reply:
👍 Fixed